src/functionalTest/resources/responses/response-solicitor-submit-application-about-to-submit.json

```diff
@@ -100,7 +100,8 @@
     ],
     "selectedDivorceCentreSiteId": "AA04",
     "dueDate": "${json-unit.any-string}",
-    "dataVersion": "${json-unit.any-number}"
+    "dataVersion": "${json-unit.any-number}",
+    "solServiceTruthStatement": "${json-unit.any-string}"
```

Comment:
solServiceTruthStatement value will always be fixed right ? so unless it is dynamic  we don't need to use `${json-unit.any-string}`